examples/decoding/plot_ssd_spatial_filters.py

```diff
@@ -183,33 +406,67 @@ def spectral_ratio_ssd(psd, freqs, freqs_sig, freqs_noise):
 # cov with the filters Eq. (22) (C_n W)^t.
 # We rely on the inversion apprpach here.
 
-
 max_idx = spec_ratio.argsort()[::-1][:4]
 layout = read_layout('CTF151.lay')
-
 pattern = mne.EvokedArray(
-    data=np.linalg.pinv(ssd.filters_)[max_idx, :].T,
-    info=mne.pick_info(raw.info, ssd.picks_))
-
+    data=ssd.patterns_[:4].T, info=mne.pick_info(raw.info, ssd.picks_))
 pattern.plot_topomap(units=dict(mag='A.U.'),
-                     time_format='', layout=layout)
-
```

Comment:
What happened to layout?

Reply:
I had before error when I passed the layout. Now seems to work. thanks to point it out :)